testutil/testclient/testeventsquery/client.go

```diff
@@ -117,3 +117,39 @@ func NewAnyTimesEventsBytesEventsQueryClient(
 
 	return eventsQueryClient
 }
+
+// NewAnyTimesClosableEventsQueryClient returns a new events query client which
+// is configured to return the expected event bytes when queried with the expected
+// query, any number of times. The returned client also expects to be closed any
+// number of times.
+func NewAnyTimesClosableEventsQueryClient(
```

Comment:
I'm not convinced that we need a new helper for this. Is there a reason that we can't add the same "any times" expectation to the existing helper?